lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1222,6 +1222,7 @@ enum SaveCoreStyle {
   eSaveCoreFull = 1,
   eSaveCoreDirtyOnly = 2,
   eSaveCoreStackOnly = 3,
+  eSaveCoreCustom = 4,
```

Comment:
Should we name this `eSaveCoreCustomOnly`? We should add some header doc for each of these above ones as well explaining what will be done and how they interact with any custom ranges that are specified. 
```
  /// Save all memory ranges that are readable to the core file. If
  /// custom memory ranges are specified, they will be ingored.
  eSaveCoreFull = 1,
  /// If the platform supports detecting dirty pages within read/write
  /// regions, save only the modified pages. If the platform doesn't
  /// support detecting dirty pages, then save all regions with read + 
  /// write permissions. If custom memory ranges are specified, they 
  /// will be merged into the regions that get saved.
  eSaveCoreDirtyOnly = 2,
  /// Save the stacks for all threads. Minimal stacks will be saved 
  /// where only the SP to the top of the stack will be saved taking
  /// into account the red zone. If custom memory ranges are specified, 
  /// they will be merged into the regions that get saved.
  eSaveCoreStackOnly = 3,
  /// Save only the custom regions that are specified. 
  eSaveCoreCustom = 4,
```

Reply:
I think `CustomOnly` sounds a bit clunky, but it does convey only custom fields will be saved and fits with the current naming standard.